### ISSUE TITLE
works first time, but says bluetooth not supported after reboot

### ISSUE THREAD

**Issue by Uddhav07** [Bananz0/GalaxyBookEnabler#24]
![image](https://github.com/user-attachments/assets/ab17bdbd-f96a-4ad7-acfd-2d9fdd279082)
samsung notes work fine
![image](https://github.com/user-attachments/assets/701793ec-4a65-4165-a381-1039de2112a3)

i might have uninstalled samsung settings by mistake

**Comment by Uddhav07** [comment]
App installation failed with error message: Windows cannot install package SAMSUNGELECTRONICSCO.LTD.SamsungSettings1.5_6.1.8.0_x64__3c1yjt4zspk6g because this package depends on a device driver that could not be found.  One of the following device drivers must be installed: {[Samsung-SFourSWComp15.inf, 6.1.0.0]} (0x80073cfd)

**Comment by Uddhav07** [comment]
just found out why this happens, it happens after a system restore, lemme check why exactly

**Comment by Uddhav07** [comment]
yeah just installed everything clean again, it works nicely then gives this error as soon as i turn it off and on anytime after installation

**Comment by andreap25** [comment]
To make Multi Control back to work, terminate all the Samsung related tasks on Task Manager, then reset Multi Control app from Windows settings. If it works, I suggest you to set up all you need in the app, like position of every device, and never open the app again in future. The devices should connect anyway even after reboot.

**Comment by Bananz0** [comment]
Thanks @andreap25 for the help. I'll close this issue if it was solved.
